src/_tutorials/server/cmdline.md

```diff
@@ -47,7 +47,7 @@ command-line arguments, files and directories, and more.
 
 ## Running an app with the standalone Dart VM
 
-To run a command-line app, you need the Dart VM (`dart`),
+To run a command-line app, you need the Dart command (`dart run`),
 which comes when you [install the Dart SDK](/get-dart).
```

Comment:
I'd like to tweak this a bit and change the SDK link:

```suggestion
To run a command-line app in the Dart VM, use `dart run`.
The `dart` command is part of the [Dart SDK](/tools/sdk).
```

Reply:
Done